app/views/clients/cas_readiness/_disability_verification.haml

```diff
@@ -1,15 +1,18 @@
+- description = GrdaWarehouse::Tasks::PushClientsToCas.new.description_display_for(:disability_verified_on)
+
 %section.o-section-card
   %h3 Disability Verification
   .c-card.c-card--block.c-card--padded.c-card--flush
+    - if description.present?
+      %p= description
```

Comment:
It felt like we should expose a description specifically on the Disability Verification card, but I'm confused as to which description should show up there.

The way it's implemented on this branch, if you attach a description to `:disability_verified_on` it shows up in two places:


<img width="538" alt="Screen Shot 2022-08-22 at 4 20 59 PM" src="https://user-images.githubusercontent.com/5333982/186012881-388f3621-17d9-4ad4-84d1-d6fa6a780084.png">


---

<img width="703" alt="Screen Shot 2022-08-22 at 4 20 40 PM" src="https://user-images.githubusercontent.com/5333982/186012883-c130ed03-10b5-4bc8-a549-6e79de2116a3.png">

Reply:
That feels right to me.  I think we may want a description column instead of dozens of tooltips.  It'll be more busy, but more clear.